### ISSUE TITLE
Dedicated view for unlisted and private videos

### ISSUE THREAD

**Issue by JohnXLivingston** [Chocobozzz/PeerTube#3415]
**Describe the problem to be solved**
For administrators that want to check if there is no problematic content on their instance, there is a checkbox for seeing unlisted and private videos on the «local» view. But theses videos are mixed with public videos. On an instance with a lot of new videos, it can be hard to follow, and be sure there is no illegal/unwanted content. 

**Describe the solution you would like:**
For administrators and moderators, it would be nice to have a new view in the left menu, that shows only local unlisted and private videos. 

**Describe alternatives you have considered**  

An other solution is to change the behaviour of the current checkbox: when checked, displays only unlisted and private videos, not public ones. 
But I think that a full view would be better. The checkbox is difficult to see for new administrators/moderators.

**Comment by test2a** [comment]
Seconded. Can we also have a functionality to disable private /unlisted videos altogether? Just like permission to upload, public instances can start small limits but no private or unlisted. Once user has passed whatever limit set by administrator, they could allow unlisted or private.

**Comment by JohnXLivingston** [comment]
> Seconded. Can we also have a functionality to disable private /unlisted videos altogether? Just like permission to upload, public instances can start small limits but no private or unlisted. Once user has passed whatever limit set by administrator, they could allow unlisted or private.

+1. Can you open another ticket for this?

**Comment by test2a** [comment]
@JohnXLivingston https://github.com/Chocobozzz/PeerTube/issues/3418 here you go

**Comment by Chocobozzz** [comment]
Implemented in https://github.com/Chocobozzz/PeerTube/commit/527a52ac4295a072927ff46761766a8b181a7603
